EIPS/eip-360.md

```diff
@@ -0,0 +1,93 @@
+---
+eip: 360
```

Comment:
```suggestion
eip: 6932
```
Assigning EIP number 6932, please also update the filename.
EIP numbers are generally the PR number.  Using 6932 as this was the earliest PR #6932

Reply:
but the current PR is 6933, will that be okay?

Reply:
Please use EIP number 6932 as this was the earliest PR.